test/lib/unit/api/api-config_spec.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const path = require('path'),
+    supertest = require('supertest'),
+    express = require('express'),
+    {Router} = express;
+
+describe('ApiConfig', () => {
```

Comment:
@ankitgyawali Each supertest promise chain needs to be closed with a:
```
.catch(done.fail);
```
otherwise error info won't be caught and reported.